itez/beneficiary/uuid.py

```diff
@@ -0,0 +1,11 @@
+import uuid
+
+
+def generate_id():
+    """
+     this function generate a 7 digit code for agent
```

Comment:
The docs sentence must be capitalized like a regular sentence and end with a fullstop

Reply:
grammar corrected